Assets/Scripts/CameraController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraController : MonoBehaviour {
+
+    //public GameObject player to hold the player Object
+    public GameObject player;
```

Comment:
Does this need to be public?

Reply:
We can also probably remove most of the comments.  I would only comment things that aren't self explanatory.